### PR TITLE
User may edit their recent check ins

### PR DESCRIPTION
**Pull request by zspencer**
Connects #191 
- [x] Write feature test
- [x] Use the extracted form in the check_ins/new view

### CODE REVIEW

File: app/controllers/check_ins_controller.rb [hunk 1]
```diff
@@ -15,6 +15,19 @@ def create
...
   end
 
+  def edit
+    @check_in = current_account.check_ins.find(params[:id])
+  end
+
+  def update
+    @check_in = current_account.check_ins.find(params[:id])
+    log_ids = check_in_params[:log_entries_attributes].map { |_, attrs| attrs[:id] }.compact
+    raise "YOU DID A BAD!" if current_account.logs.where(id: log_ids).count != log_ids.count
```
Comment (zspencer) [inline app/controllers/check_ins_controller.rb]: Let's extract these two lines to their own method: `authorize_current_account_may_update_log_entries`

Comment (Wimsy113) [inline app/controllers/check_ins_controller.rb]: which two lines? lol

Comment (zspencer) [inline app/controllers/check_ins_controller.rb]: 24 and 25.

File: app/models/check_in.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@ def log_entries_attributes=(log_entries_attributes)
 
   def log_entries
     Activity.active.map do |activity|
```
Comment (zspencer) [inline app/models/check_in.rb:17]: We want to restrict this to the user for the check-in's active activities; not _all_ the activities.

**Pull request merged**